### ISSUE TITLE
SecurityIT.reload_when_anonymous_session_expires is flaky

### ISSUE THREAD

**Issue by mcollovati** [vaadin/flow#13443]
### Description of the bug / feature

Sometimes it fails because of unexpected HTTP response code 401 instead of 200 (1), other times it complains about `unknown error: unhandled inspector error: {"code":-32000,"message":"Execution context was destroyed."}` (2).

```
[pool-1-thread-3] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-06T11:50:19.825Z] [SEVERE] http://localhost:9999/connect/PublicEndpoint/getServerTime - Failed to load resource: the server responded with a status of 401 ()'
[pool-1-thread-5] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-06T11:50:21.067Z] [SEVERE] http://localhost:9999/connect/BalanceEndpoint/getBalance - Failed to load resource: the server responded with a status of 401 ()'
[pool-1-thread-5] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-06T11:50:21.084Z] [SEVERE] http://localhost:9999/vaadin/VAADIN/build/vaadin-bundle-862603a5319aabd31363.cache.js 543:8955 Uncaught T: expected "200 OK" response, but got 401 '
[pool-1-thread-3] INFO com.vaadin.testbench.ScreenshotOnFailureRule - Error screenshot written to: /home/runner/work/flow/flow/flow-tests/vaadin-spring-tests/test-spring-security-fusion-urlmapping/error-screenshots/reload_when_anonymous_session_expires[ANY_Chrome_](com.vaadin.flow.spring.fusionsecuritycontextpath.SecurityIT).png
[pool-1-thread-2] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-06T11:50:17.951Z] [SEVERE] http://localhost:9999/admin-only/secret.txt - Failed to load resource: the server responded with a status of 403 ()'
[pool-1-thread-2] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-06T11:50:17.996Z] [SEVERE] http://localhost:9999/admin-only/secret.txt - Failed to load resource: the server responded with a status of 403 ()'
Error:  Tests run: 17, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 22.17 s <<< FAILURE! - in com.vaadin.flow.spring.fusionsecuritycontextpath.SecurityIT
Error:  reload_when_anonymous_session_expires[ANY_Chrome_]  Time elapsed: 5.792 s  <<< ERROR!
org.openqa.selenium.JavascriptException: 
javascript error: expected "200 OK" response, but got 401 
  (Session info: headless chrome=100.0.4896.60)
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'fv-az136-213', ip: '10.1.1.36', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-1017-azure', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [63ebb9f8dc895cb5ca3346e200e41a4d, executeScript {script=return arguments[0].updateTime(), args=[{ELEMENT=eaa4208b-d077-4110-8fd9-021ea27a2541, element-6066-11e4-a52e-4f735466cecf=eaa4208b-d077-4110-8fd9-021ea27a2541}]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.60, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: /tmp/.com.google.Chrome.hqaeTi}, goog:chromeOptions: {debuggerAddress: localhost:46339}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:46339/devtoo..., se:cdpVersion: 100.0.4896.60, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 63ebb9f8dc895cb5ca3346e200e41a4d
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:558)
	at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:492)
	at com.vaadin.testbench.TestBenchDriverProxy.executeScript(TestBenchDriverProxy.java:184)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvstf99_0._d7executeScript(TestBenchDriverProxy_$$_jvstf99_0.java)
	at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.vaadin.testbench.DriverInvocationHandler.invoke(DriverInvocationHandler.java:48)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvstf99_0.executeScript(TestBenchDriverProxy_$$_jvstf99_0.java)
	at com.vaadin.testbench.commands.TestBenchCommandExecutor.executeScript(TestBenchCommandExecutor.java:249)
	at com.vaadin.testbench.TestBenchElement.executeScript(TestBenchElement.java:700)
	at com.vaadin.testbench.TestBenchElement_$$_jvstf99_1._d15executeScript(TestBenchElement_$$_jvstf99_1.java)
	at jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.vaadin.testbench.TestBench$ElementInvocationHandler.invoke(TestBench.java:78)
	at com.vaadin.testbench.TestBenchElement_$$_jvstf99_1.executeScript(TestBenchElement_$$_jvstf99_1.java)
	at com.vaadin.testbench.TestBenchElement.callFunction(TestBenchElement.java:712)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.assertPublicEndpointReloadsPage(SecurityIT.java:398)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.reload_when_anonymous_session_expires(SecurityIT.java:269)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at com.vaadin.testbench.parallel.ParallelRunner$TBMethod.invokeExplosively(ParallelRunner.java:491)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.vaadin.testbench.parallel.ParallelRunner$1.evaluate(ParallelRunner.java:465)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at com.vaadin.testbench.parallel.ParallelScheduler$1.call(ParallelScheduler.java:44)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

```
pool-1-thread-3] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-07T07:02:49.117Z] [SEVERE] http://localhost:9999/vaadin/VAADIN/build/vaadin-bundle-7b26b38808438e9c1222.cache.js 543:8955 Uncaught T: expected "200 OK" response, but got 401 '
[pool-1-thread-5] INFO com.vaadin.testbench.ScreenshotOnFailureRule - Error screenshot written to: /home/runner/work/flow/flow/flow-tests/vaadin-spring-tests/test-spring-security-fusion-urlmapping/error-screenshots/reload_when_anonymous_session_expires[ANY_Chrome_](com.vaadin.flow.spring.fusionsecuritycontextpath.SecurityIT).png
[pool-1-thread-2] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-07T07:02:46.887Z] [SEVERE] http://localhost:9999/admin-only/secret.txt - Failed to load resource: the server responded with a status of 403 ()'
[pool-1-thread-2] WARN com.vaadin.flow.testutil.TestBenchHelpers - This message in browser log console may be a potential error: '[2022-04-07T07:02:46.914Z] [SEVERE] http://localhost:9999/admin-only/secret.txt - Failed to load resource: the server responded with a status of 403 ()'
Error:  Tests run: 17, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 17.538 s <<< FAILURE! - in com.vaadin.flow.spring.fusionsecuritycontextpath.SecurityIT
Error:  reload_when_anonymous_session_expires[ANY_Chrome_]  Time elapsed: 4.076 s  <<< ERROR!
org.openqa.selenium.WebDriverException: 
unknown error: unhandled inspector error: {"code":-32000,"message":"Execution context was destroyed."}
  (Session info: headless chrome=100.0.4896.60)
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'fv-az119-192', ip: '10.1.0.34', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-1017-azure', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8aa6e954a3c926caa9e19a91e26c7bec, executeScript {script=return arguments[0].updateTime(), args=[{ELEMENT=84fc12be-1eed-4c48-a601-96955ecf10c8, element-6066-11e4-a52e-4f735466cecf=84fc12be-1eed-4c48-a601-96955ecf10c8}]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.60, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: /tmp/.com.google.Chrome.pAPLt5}, goog:chromeOptions: {debuggerAddress: localhost:34725}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:34725/devtoo..., se:cdpVersion: 100.0.4896.60, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 8aa6e954a3c926caa9e19a91e26c7bec
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:558)
	at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:492)
	at com.vaadin.testbench.TestBenchDriverProxy.executeScript(TestBenchDriverProxy.java:184)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvst396_0._d7executeScript(TestBenchDriverProxy_$$_jvst396_0.java)
	at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.vaadin.testbench.DriverInvocationHandler.invoke(DriverInvocationHandler.java:48)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvst396_0.executeScript(TestBenchDriverProxy_$$_jvst396_0.java)
	at com.vaadin.testbench.commands.TestBenchCommandExecutor.executeScript(TestBenchCommandExecutor.java:249)
	at com.vaadin.testbench.TestBenchElement.executeScript(TestBenchElement.java:700)
	at com.vaadin.testbench.TestBenchElement_$$_jvst396_1._d15executeScript(TestBenchElement_$$_jvst396_1.java)
	at jdk.internal.reflect.GeneratedMethodAccessor11.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.vaadin.testbench.TestBench$ElementInvocationHandler.invoke(TestBench.java:78)
	at com.vaadin.testbench.TestBenchElement_$$_jvst396_1.executeScript(TestBenchElement_$$_jvst396_1.java)
	at com.vaadin.testbench.TestBenchElement.callFunction(TestBenchElement.java:712)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.assertPublicEndpointReloadsPage(SecurityIT.java:398)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.reload_when_anonymous_session_expires(SecurityIT.java:269)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at com.vaadin.testbench.parallel.ParallelRunner$TBMethod.invokeExplosively(ParallelRunner.java:491)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.vaadin.testbench.parallel.ParallelRunner$1.evaluate(ParallelRunner.java:465)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at com.vaadin.testbench.parallel.ParallelScheduler$1.call(ParallelScheduler.java:44)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```	

### Versions:
    - Vaadin / Flow version: 23.1-SNAPSHOT
    - Java version: 11
    - OS version: Ubuntu 21.10

**Comment by Artur-** [comment]
This is what #13436 should fix

**Comment by mcollovati** [comment]
PR vaadin/flow#13449 has the patch, but it seems it still fails, perhaps for other reasons.

```
Error:  Tests run: 17, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 17.192 s <<< FAILURE! - in com.vaadin.flow.spring.fusionsecuritycontextpath.SecurityIT
Error:  reload_when_anonymous_session_expires[ANY_Chrome_]  Time elapsed: 4.075 s  <<< ERROR!
org.openqa.selenium.StaleElementReferenceException: 
stale element reference: element is not attached to the page document
  (Session info: headless chrome=100.0.4896.60)
For documentation on this error, please visit: https://selenium.dev/exceptions/#stale_element_reference
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'fv-az96-801', ip: '10.1.0.143', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-1017-azure', java.version: '17.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [3bb44888465a26010994577873c3d01d, findChildElement {id=018ccf28-83f6-4115-acd9-c5691e4f4c81, using=id, value=time}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 100.0.4896.60, chrome: {chromedriverVersion: 100.0.4896.60 (6a5d10861ce8..., userDataDir: /tmp/.com.google.Chrome.orDUMf}, goog:chromeOptions: {debuggerAddress: localhost:33069}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:33069/devtoo..., se:cdpVersion: 100.0.4896.60, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 3bb44888465a26010994577873c3d01d
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:558)
	at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
	at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:382)
	at org.openqa.selenium.remote.RemoteWebElement.findElement(RemoteWebElement.java:221)
	at com.vaadin.testbench.TestBenchElement.findElement(TestBenchElement.java:276)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.assertPublicEndpointReloadsPage(SecurityIT.java:397)
	at com.vaadin.flow.spring.fusionsecurity.SecurityIT.reload_when_anonymous_session_expires(SecurityIT.java:268)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at com.vaadin.testbench.parallel.ParallelRunner$TBMethod.invokeExplosively(ParallelRunner.java:491)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.vaadin.testbench.parallel.ParallelRunner$1.evaluate(ParallelRunner.java:465)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at com.vaadin.testbench.parallel.ParallelScheduler$1.call(ParallelScheduler.java:44)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

[INFO] 
[INFO] Results:
[INFO] 
Error:  Errors: 
Error:    SecurityIT>SecurityIT.reload_when_anonymous_session_expires:268->SecurityIT.assertPublicEndpointReloadsPage:397 » StaleElementReference
```

**Comment by Artur-** [comment]
The problem is likely then that a page reload happens between `getPublicView()` and `.findElement(...)`. Another fix in #13466
